bitvm/src/bn254/g1.rs

```diff
@@ -305,9 +306,9 @@ impl G1Affine {
 
     
     fn hinted_scalar_decomposition(k: ark_bn254::Fr) -> (Script, Vec<Hint>) {
-        const LAMBDA: ark_bn254::Fr = MontFp!("21888242871839275217838484774961031246154997185409878258781734729429964517155");
+        let lambda: ark_bn254::Fr = ark_bn254::Fr::from(BigUint::from_str("21888242871839275217838484774961031246154997185409878258781734729429964517155").unwrap());
```

Comment:
`Lambda` should stay a const.

Reply:
if it's okay, i'd like to defer this to a cleanup PR after this one.
- I get a rust-analyzer error: "proc macro panicked: could not parse" whenever I use MontFp macro. Build and runs works just fine, only a problem with rust-analyzer. 
- Secondly assigning const to value returned by Fr::from() doesn't seem possible at the moment.
```
cannot call non-const fn `<ark_ff::Fp<MontBackend<FrConfig, 4>, 4> as From<BigUint>>::from` in constants
calls in constants are limited to constant functions, tuple structs and tuple variants
```
There are few other instances too where const should be used. Will do them in batch in cleanup PR if okay.